### ISSUE TITLE
Failed to import python plugin: community/borzacchiello_seninja

### ISSUE THREAD

**Issue by ysf** [borzacchiello/seninja#4]
Hey there, 

I tried to install seninja using Version 2.2.2489-dev Personal of binary ninja. Unfortunately it hat problems starting:

```
Failed to import python plugin: community/borzacchiello_seninja: No module named 'seninja'
Failed to enable plugin:
SENinja
Check the log for more details
```

Any idea how this happens? Thanks a lot for your work, really eager to use it!

**Comment by borzacchiello** [comment]
Hi, thanks for using the plugin!
Try renaming the folder from `borzacchiello_seninja` to `seninja`. It should work but it's a hack. I messed up with some import probably, I'll look into it shortly.

**Comment by borzacchiello** [comment]
The problem should be solved in version 0.2.2, just updated in the plugin manager. Let me know if it works now.
